Tests/Opc.Ua.Client.Tests/ClientFixture.cs

```diff
@@ -283,7 +284,28 @@ public ISession CreateSession(ITransportChannel channel, ConfiguredEndpoint endp
         {
             if (endpoints == null)
             {
-                endpoints = await GetEndpoints(url).ConfigureAwait(false);
+                try
+                {
+                    endpoints = await GetEndpoints(url).ConfigureAwait(false);
+                }
+                catch(Exception)
+                {
+                    // Don't ignore if platform is Windows
+                    if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                    {
+                        throw;
+                    }
+                    else
+                    {
+                        // Don't ignore schemes other than opc.https or https
+                        if (!url.Scheme.Equals("opc.https", StringComparison.OrdinalIgnoreCase) &&
+                            !url.Scheme.Equals("https", StringComparison.OrdinalIgnoreCase))
+                        {
```

Comment:
these strings exist as constants in core, please use them instead of writing it out